### ISSUE TITLE
Can I call swapSuggestions method with no animation?

### ISSUE THREAD

**Issue by juancitogo** [arimorty/floatingsearchview#148]
When I want to update suggestions list with the new result set and call swapSuggestions method, the list performs an animation when is showing the new results, but with my large suggestions list I want to use no animation.

Is there a way to set up the floating search view to do the swap suggestions with no animation?

Thanks in advance,

**Comment by arimorty** [comment]
Hi @juancitogo ,

You can simply set the animation duration to 0 in code: `mSearchView.setSuggestionsAnimDuration(0);` or
 in xml: `app:floatingSearch_suggestionsListAnimDuration="0"`

Regards,
Ari

**Comment by juancitogo** [comment]
Hi @arimorty ,

thanks for your help.

I test it with both methods but animation on suggestion list is still running.

In xml:

```
<com.arlib.floatingsearchview.FloatingSearchView
        android:id="@+id/floating_search_view"
        android:layout_width="match_parent"
        android:layout_height="match_parent"
        app:floatingSearch_dismissOnOutsideTouch="true"
        app:floatingSearch_leftActionMode="showHome"
        app:floatingSearch_searchBarMarginLeft="@dimen/search_view_inset"
        app:floatingSearch_searchBarMarginRight="@dimen/search_view_inset"
        app:floatingSearch_searchBarMarginTop="@dimen/search_view_inset"
        app:floatingSearch_searchHint="Search..."
        app:floatingSearch_showSearchKey="false"
        app:floatingSearch_menu="@menu/menu_search_view"
        app:floatingSearch_suggestionsListAnimDuration="0"/>
```

and using method:

`mSearchView = (FloatingSearchView) findViewById(R.id.floating_search_view);
        mSearchView.setSuggestionsAnimDuration(0);`

Is there any other attribute that I need to set?

Thanks,

**Comment by 1240** [comment]
+1, disable animation not work

**Comment by juancitogo** [comment]
Hi @arimorty 

setting the animation duration to 0 works on android version superior to 5.1.1

do you plan some fix for versions 5.1.1 and lower?

thanks,

**Comment by arimorty** [comment]
@juancitogo The animation your are referring to is the a result of a list scroll. The next version of the library has significant changes put in place in order to avoid that scroll.

Thanks for contributing!

Best,
Ari
